db/migrate/20181125004524_create_executions.rb

```diff
@@ -0,0 +1,15 @@
+class CreateExecutions < ActiveRecord::Migration[5.2]
+  def change
+    create_table :executions do |t|
+      t.integer :script_id, null: false, index: true
```

Comment:
Ahora entendí porque pasaba lo de abajo:
```suggestion
      t.references :script, index: true, null: false
```